src/main/resources/com/welovecoding/nbeditorconfig/EditorConfigExample.editorconfig

```diff
@@ -0,0 +1,33 @@
+# EditorConfig is awesome: http://EditorConfig.org
```

Comment:
Good to have a detailed example config! We also have an `.editorconfig` file in `src\main\resources\com\welovecoding\netbeans\plugin\editorconfig\filetype\.editorconfig` which is used as a template for new files. We should remove our file and take your new one to only have one template. :)